licences-specs/src/test/groovy/uk/gov/justice/digital/hmpps/licences/specs/postDecision/CaTaskListSpec.groovy

```diff
@@ -79,6 +81,21 @@ class CaTaskListSpec extends GebReportingSpec {
 
     and: 'create licence is available'
     taskListAction(tasks.create).text() == 'Continue'
+
+    and: 'Resubmit to DM is available'
+    taskListAction(tasks.resubmitToDm).text() == 'Resubmit'
+  }
+
+  def 'Shows Send Approval page'() {
+
+    when: 'I view the task list'
+    to TaskListPage, testData.markAndrewsBookingId
+
+    then: 'I click the Resubmit button'
+    taskListActions[10].click()
```

Comment:
When writing web integration tests the idea is to not bind directly to the structure/layout of the page. This hardcodes the order of the actions which isn't great - would be a more robust test if the action was identified  based on name or even better some `data-qa` attribute